crates/sui/scripts/prover_run.sh

```diff
@@ -0,0 +1,16 @@
+#  Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
Are we supposed to have the year here too? Other existing files seem to be just "Copyright (c) Mysten Labs, Inc."

Reply:
Nope, fixing!